### PR TITLE
Feat/250 confirmation dialog

### LINKED ISSUE #250: [Feature flipping] Confirmation dialog when turning off feature
Hi,
Turning off a feature can be very sensitive. However, accidentally switching the wrong button is pretty easy (especially on a touch screen !). 
One solution to this would be to open an alert dialog when turning a feature off. 
windowdialog

Ideally this would be activated through the conf files (maybe activated by default in production).

What do you think about this ?

### PR DESCRIPTION
**Pull request by cbm-bhoux**
Fix #250 

Added a configuration file for front-end, is it fine this way ?

### CODE REVIEW

**Comment by larousso** [comment]
Hi, 

The way to do this is to add a property on the _env object and pass it to the view next to auth :

https://github.com/MAIF/izanami/blob/master/izanami-server/app/controllers/HomeController.scala#L32

and use it here as it'is done for the user : 

https://github.com/MAIF/izanami/blob/master/izanami-server/app/views/index.scala.html#L27

then you can use it in the react code 

This config should come from the `env.IzanamiConfig` that is parsed from the `application.conf` file using the "pure config" lib. 
Basically you need to add an entry in `application.conf` like `confirmationDialog = false` and the associated environment variable like it's done for the `logout` property. 
Then in the configuration.scala file you add the entry in `IzanamiConfig`. 
In the `HomeController` you can get this property and use it.

I hope this is clear (but I would not bet :) )

**Comment by cbm-bhoux** [comment]
Hi, thanks for explaining everything ! I added confirmationDialog to the conf, let me know if this is better :)

**Comment by larousso** [comment]
Thanks for your contribution !
